az_quickstart.install

```diff
@@ -195,3 +195,10 @@ function az_quickstart_update_1020802() {
 function az_quickstart_update_1020803() {
   \Drupal::service('module_installer')->uninstall(['media_library_theme_reset']);
 }
+
+/**
+ * Enable ckeditor_indentblock module by default.
+ */
+function az_quickstart_update_1021001() {
+  \Drupal::service('module_installer')->install(['ckeditor_indentblock']);
+}
```

Comment:
Should we move this to `az_paragraphs.install` since the module dependency is being added there?

Reply:
I like that idea